logic/settings.ts

```diff
@@ -22,6 +22,7 @@ export enum CountType {
 	Created = "created",
 	Modified = "modified",
 	FileSize = "filesize",
+	CustomKey = "customkey",
```

Comment:
I think a more descriptive name would be:

```suggestion
	FrontmatterKey = "frontmatterkey",
```